en/simulation/hitl.md

```diff
@@ -20,17 +20,38 @@ Airframe | `SYS_AUTOSTART` | X-Plane | Gazebo | jMAVSim
 
 ## HITL Simulation Environment {#simulation_environment}
 
-With Hardware-in-the-Loop (HITL) simulation the normal PX firmware is run on real hardware. *QGroundControl* is connected to the physical hardware over USB and acts as a gateway to forward data between the simulator running on a development computer, PX4, and any offboard API.
+With Hardware-in-the-Loop (HITL) simulation the normal PX firmware is run on real hardware. The HITL configuration is slightly different for Gazebo, jMAVSim and X-Plane.
+
+> **Tip** For more information see: [Simulation](../simulation/README.md).
+
+### JMAVSim/Gazebo HITL Environment
+
+JMAVSim or Gazebo (running on a development computer) are connected to the flight controller hardware via USB/UART. The simulator acts as gateway to share MAVLink data between PX4 and *QGroundControl*. 
+
+> **Tip** Gazebo can additionally share MAVLink data with an offboard API!
+
+The diagram below shows the simulation environment:
+* A HITL configuration is selected (via *QGroundControl*) that doesn't start any real sensors.
+* *jMAVSim* or *Gazebo* are connected to the flight controller via USB.
```

Comment:
I just remembered that there is another possibility: you can do HIL via Ethernet (or even WiFi, but you need a good one with low latency). I've done that on the Raspberry Pi. This ist the startup configuration, including how to run jMAVSim:
https://github.com/PX4/Firmware/blob/master/posix-configs/rpi/px4_hil.config
fyi @julianoes

Reply:
Thanks, I have fixed typos and merged as is. Created separate issue for https://github.com/PX4/Devguide/issues/466